Cubical/Categories/CBPV/Instances/SmallStep.agda

```diff
@@ -0,0 +1,94 @@
+{-# OPTIONS --lossy-unification #-}
+
+module Cubical.Categories.CBPV.Instances.SmallStep where
+    open import Cubical.Categories.Instances.Sets
```

Comment:
1. We don't indent inside of the top-level module
2. We use 2-space indentation, not 4.